dashboard/UI-Test/selenium_drivers/README.md

```diff
@@ -0,0 +1,9 @@
+Driver version currently shipped:
+- Chrome/Chromium = v105.0.5195.19
+- Firefox(gecko) = v0.31.0
+
+> NOTE:
```

Comment:
As I commented above, this is an "interesting" use of block quote, which I suggest replacing with bold and/or italics or a four/five-level header.